lifecycle-management/Ahk.Lifecycle.Management/Data/RepositoryStatistics.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace Ahk.Lifecycle.Management
+{
+    public class RepositoryStatistics
+    {
+        [Key]
+        public string Repository { get; set; }
```

Comment:
erre ügyelni kell, hogy normalizált legyen, kisbetű-nagybetű eltérés ne okozzon problémát, mindig legyen kisbetűs, amikor elmentésre kerül, lásd itt: https://github.com/akosdudas/ahk-github-automation/blob/master/grade-management/Ahk.GradeManagement.Data/Normalize.cs

Reply:
Át lett nevezve Id-ra. CosmosClient beállításaiból adódóan kis betűs lesz (mint minden más).